src/azure-cli/azure/cli/command_modules/acr/commands.py

```diff
@@ -237,6 +245,11 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
         g.command('logs', 'acr_task_logs', client_factory=cf_acr_runs,
                   table_transformer=None)
 
+    with self.command_group('acr taskrun', acr_taskrun_util) as g:
+        g.command('list', 'acr_taskrun_list')
+        g.command('delete', 'acr_taskrun_delete')
+        g.command('show', 'acr_taskrun_show')
```

Comment:
It just occurs to me if we can introduce `az taskrun logs`, it will be very handy. So users don't have to pipe the runid from `az taskrun show` to `az task logs`. You may consider in future pr.